tutorials/misc/gles2_gles3_differences.rst

```diff
@@ -196,6 +196,24 @@ For a complete list of built-in GLSL functions see the :ref:`Shading Language do
 | vec_type **fwidth** ( vec_type )                                       |                                                  |
 +------------------------------------------------------------------------+--------------------------------------------------+
 
+**textureSize() workaround**
```

Comment:
This should use an rst style heading. 

I suggest

textureSize() workaround
^^^^^^^^^^^^^^^^^^^^^^^